docs/spark/spark-writes.md

```diff
@@ -328,14 +330,32 @@ PARTITIONED BY (days(ts), category)
 
 To write data to the sample table, your data needs to be sorted by `days(ts), category`.
 
-If you're inserting data with SQL statement, you can use `ORDER BY` to achieve it, like below:
+If you're inserting data with SQL statement, you can use either `ORDER BY` to trigger global sort, or `SORT BY` to trigger local sort. Global sort like below:
 
 ```sql
 INSERT INTO prod.db.sample
 SELECT id, data, category, ts FROM another_table
 ORDER BY ts, category
 ```
 
+Local sort like below:
+
+```sql
+INSERT INTO prod.db.sample
+SELECT id, data, category, ts FROM another_table
+SORT BY ts, category
+```
+
+Of course, you can also use `SORT BY` with partition transformations. 
```

Comment:
Pronouns here
"Of Course" I think is a bit colloquial and doesn't add much here

Instead 
"SORT BY clauses can also be used with partition transforms" 

I think this also probably should be accompanied by an example ^ since I don't think a novice reader would understand quite what this means.